modules/ocf_printhost/files/enforcer

```diff
@@ -147,19 +160,29 @@ def send_printer_mail(message, job, quo):
     send_mail_user(job.user, message.subject, body)
 
 
-def prehook(c, job):
+def prehook(c, r, job):
     quo = quota.get_quota(c, job.user)
     if job.pages > quo.daily:
         send_printer_mail(INSUFFICIENT_QUOTA_MESSAGE, job, quo)
+        msg = NOTIFY_QUOTA_MESSAGE.format(
+            pages=job.pages,
+            quota=quo.daily)
+        r.publish(job.user, msg)
         sys.exit(255)
 
 
-def posthook(c, job, success):
+def posthook(c, r, job, success):
+    msg = ''
     if success:
         quota.add_job(c, job)
+        msg = NOTIFY_JOB_ACCEPTED.format(
+            document=job.doc_name,
+            printer=job.printer)
```

Comment:
For this (and the one above this), I'd format it like this:

```
msg = NOTIFY_JOB_ACCEPTED.format(
    document=job.doc_name,
    printer=job.printer,
)
```

Then it's clear where the statement ends and you get a trailing comma!